packages/graphql-transformers-e2e-tests/src/__tests__/ModelAuthTransformer.e2e.test.ts

```diff
@@ -528,6 +528,7 @@ describe(`ModelAuthTests`, async () => {
     );
     expect(updateResponse.data.updatePost).toEqual(null);
     expect(updateResponse.errors.length).toEqual(1);
+    expect((updateResponse.errors[0] as any).data).toBeNull();
```

Comment:
As any >.<

Reply:
Good point here there is a few instances of this in some of the tests, though this change might be best in it's own PR.